aclk/agent_cloud_link.h

```diff
@@ -73,6 +73,7 @@ void *aclk_main(void *ptr);
       .start_routine = aclk_main },
 
 extern int aclk_send_message(char *sub_topic, char *message, char *msg_id);
+extern void aclk_lws_wss_mqtt_layer_disconect_notif();
```

Comment:
why not just include the file? That way we don't have 3 places to update if function parameters change in future

Reply:
> why not just include the file? That way we don't have 3 places to update if function parameters change in future

Added